### PR TITLE
refactor(location)!: remove v8 deprecated location methods

### PR DESCRIPTION
**Pull request by Shinigami92**

### CODE REVIEW

File: test/modules/helpers.spec.ts [hunk 1]
```diff
@@ -1038,9 +1038,6 @@ describe('helpers', () => {
         });
 
         it('should support deprecated module aliases', () => {
-          expect(faker.definitions.location.city_name).toContain(
-            faker.helpers.fake('{{address.cityName}}')
```
Comment (xDivisionByZerox) [inline test/modules/helpers.spec.ts:1042]: Can we test another address fake pattern here to ensure that it keeps working?

Comment (Shinigami92) [reply test/modules/helpers.spec.ts:1042]: Please suggest one

Comment (xDivisionByZerox) [reply test/modules/helpers.spec.ts:1042]: I am currently on my phone. But I can do that when I'm off work.

Comment (xDivisionByZerox) [reply test/modules/helpers.spec.ts:1042]: GitHub doesn't allow me to make a suggestion on deleted lines. You can use `faker.helpers.fake('{{address.state}}')`, tho.

**Review by ST-DDT — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]

**Pull request merged**